scripts/githooks/pre-push

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+
+require 'net/http'
+require 'uri'
+require 'json'
+
+branch_name = `git rev-parse --abbrev-ref HEAD`
+jira_pattern = /[A-Z]+-\d+/i
+jira_branch_match = jira_pattern.match(branch_name)
+
+if jira_branch_match
+  branch_issue = jira_branch_match[0].upcase
+  jira_creds_dir = File.join(ENV['HOME'], '.jira', 'credentials')
+
+  if File.exist?(jira_creds_dir)
+    creds = Hash[File.read(jira_creds_dir).split("\n").map{|i|i.split('=')}]
```

Comment:
What would happen if they had this file but had it formatted incorrectly causing this to create a bad (unintended) hash

Reply:
I think it would be a pretty obvious error when they try to push, running an `npm i` will fix it once formatted correctly.. I'm more concerned about catching Http errors-- will see how bad it is when people start using the hook (can roll people on slowly)